src/test/java/io/nats/client/impl/JetStreamTestBase.java

```diff
@@ -44,22 +45,35 @@ public class JetStreamTestBase extends TestBase {
 
     public static final Duration DEFAULT_TIMEOUT = Duration.ofMillis(1000);
 
+    private static final AtomicInteger MOCK_SID_HOLDER = new AtomicInteger(4273);
+    public static String mockSid() {
+        return "" + MOCK_SID_HOLDER.incrementAndGet();
+    }
+
```

Comment:
because the sid check is built in to the push message manager, all mocked messages must have a good and knowable sid.